.github/workflows/docs.yml

```diff
@@ -71,6 +71,6 @@ jobs:
         # This is NOT recommended, as it can break your workflows without notice!
         # We recommend specifying a fixed version, i.e. @8.0.0
         # Docs: https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions
-        uses: readmeio/rdme@main
+        uses: readmeio/rdme@next
         with:
-          rdme: docs ./documentation --key=${{ secrets.README_DEVELOPERS_API_KEY }} --version=${{ vars.README_DEVELOPERS_MAIN_VERSION }}
+          rdme: docs upload./documentation --key=${{ secrets.README_DEVELOPERS_API_KEY }} --version=${{ vars.README_DEVELOPERS_MAIN_VERSION }}
```

Comment:
```suggestion
          rdme: docs upload ./documentation --key=${{ secrets.README_DEVELOPERS_API_KEY }} --version=${{ vars.README_DEVELOPERS_MAIN_VERSION }}
```

Reply:
Lol dammit that's what I get for trying to wrap up everything before the end of the day on a friday